python-sdk/src/astro/files/locations/ftp/ftp.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from urllib.parse import urlparse
+
+from airflow.providers.ftp.hooks.ftp import FTPHook
+
+from astro.constants import FileLocation
+from astro.files.locations.base import BaseFileLocation
+
+
+class FTPLocation(BaseFileLocation):
+    """Handler FTP object store operations"""
+
+    location_type = FileLocation.FTP
+
+    @property
+    def hook(self) -> FTPHook:
+        return FTPHook(ftp_conn_id=self.conn_id) if self.conn_id else FTPHook()
+
+    @property
+    def transport_params(self) -> dict:
```

Comment:
Is this always going to be empty dict?

Reply:
maybe worth have a loading ftp file in Postgres to test smart open path

Reply:
Yes since we pass the details in URI along with path